contrib/Cargo.toml

```diff
@@ -37,5 +42,11 @@ handlebars = { version = "0.32", optional = true }
 glob = { version = "^0.2", optional = true }
 tera = { version = "0.11", optional = true }
 
+# Database dependencies
+diesel = { version = "1.1", default-features = false, optional = true }
```

Comment:
Do we need anything in `1.1`, or will using `1.0` suffice? Better to be lenient if possible.

Reply:
We don't use anything in 1.1 that I know of. I'll drop the dependency to be just 1.0 to be as lenient as possible.